doc/man7/provider-base.pod

```diff
@@ -220,22 +220,52 @@ useless without at least provider_query_operation(), and
 provider_gettable_params() is fairly useless if not accompanied by
 provider_get_params().
 
+=head2 Provider parameters
+
+provider_get_params() must return the following provider parameters to the core.
+
+=over 4
+
+=item "name" (B<OSSL_PROV_PARAM_NAME>) <UTF8_ptr>
+
+This points to a string that should give a unique description of the provider.
```

Comment:
s/description of/name for/

Reply:
I wonder if these is a good idea.  A provider author can't necessarily know what other providers there are.  It would probably make more sense to use the module full path on the libcrypto side of things.